libs/sdk-ui-ext/src/internal/dashboardEmbedding/ScheduledMail/ScheduledMailDialog/RecipientsSelect/RecipientsSelect.tsx

```diff
@@ -0,0 +1,95 @@
+// (C) 2019-2020 GoodData Corporation
+/* eslint-disable import/named,import/namespace */
+import React, { useMemo, useState } from "react";
+import { GoodDataSdkError } from "@gooddata/sdk-ui";
+import { IAnalyticalBackend } from "@gooddata/sdk-backend-spi";
+import sortBy from "lodash/sortBy";
+
+import { useWorkspaceUsers } from "../../../hooks/useWorkspaceUsers";
+import { IScheduleEmailRecipient } from "../../interfaces";
+
+import { RecipientsSelectRenderer } from "./RecipientsSelectRenderer";
+
+export interface IRecipientsSelectProps {
+    /**
+     * Author of the scheduled email - is always recipient of the scheduled email.
+     */
+    currentUser: IScheduleEmailRecipient;
+
+    /**
+     * Currently selected recipients.
+     */
+    value: IScheduleEmailRecipient[];
+
+    /**
+     * Callback to be called, when recipients are changed.
+     */
+    onChange: (recipientEmails: IScheduleEmailRecipient[]) => void;
+
+    /**
+     * Callback to be called, when error occurs when loading the recipients.
+     */
+    onError: (error: GoodDataSdkError) => void;
+
+    /**
+     * Has user canListUsersInProject permission?
+     */
+    canListUsersInProject: boolean;
+
+    /**
+     * Is enableKPIDashboardScheduleRecipients feature flag turned on?
+     */
+    enableKPIDashboardScheduleRecipients: boolean;
+
+    /**
+     * Backend to work with.
+     *
+     * Note: the backend must come either from this property or from BackendContext. If you do not specify
+     * backend here, then the executor MUST be rendered within an existing BackendContext.
```

Comment:
executor is misleading here, what about component? Ditto for workspace

Reply:
Right, some copy paste, I'll update it.